.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,23 @@
+name: Run all tests on CPU (GitHub-hosted)
+
+on: push
+
+jobs:
+  run_tests_torch_and_tf:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        pip install .[sklearn,tf,torch,testing]
+
+    - name: Run all tests on CPU
+      env:
+        OMP_NUM_THREADS: 1
+      run: |
+        python -m pytest -n 8 --dist=loadfile -s -v ./tests/
+
```

Comment:
To benchmark vs. Circle-CI, I've added this one (runs on a GitHub instance)

Seems [almost 3x slower](https://github.com/huggingface/transformers/runs/468406732?check_suite_focus=true) than the machines we have in our Circle CI plan (ofc, it's free vs. we pay Circle CI)

Reply:
One of the advantage I see using GitHub Actions rather than CircleCI is the ability to dig faster and easier in case on failure. The GA interface is more clear and you can spot directly what was the issue.